### PR TITLE
jellyfin-mpv-shim: 1.10.4 -> 2.0.1

### PR DESCRIPTION
**Pull request by jojosch**
###### Motivation for this change
Update to latest release https://github.com/jellyfin/jellyfin-mpv-shim/releases/tag/v2.0.0 [(changes)](https://github.com/jellyfin/jellyfin-mpv-shim/compare/v1.10.4...v2.0.0)

Upstream removed the desktop mode, since now there is the new `jellyfin-media-player` as a replacement for the desktop mode. Should there be a notice / changelog for that? The desktop mode was not in a stable channel.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
```
/nix/store/y06995wa30h49rir7zwpy84a35js3ckp-jellyfin-mpv-shim-1.10.4	  953988312
/nix/store/9pvi8dcpf0d5fqlbg3zfs26kvnwfrm4y-jellyfin-mpv-shim-2.0.0	  904213992
/nix/store/iaamjq35jhdpk8nl15mk2k2kmaamzcr0-jellyfin-mpv-shim-2.0.1	  901268936
```
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
> Upstream removed the desktop mode, since now there is the new jellyfin-media-player as a replacement for the desktop mode. Should there be a notice / changelog for that?

Don't think we need that.

File: pkgs/applications/video/jellyfin-mpv-shim/default.nix [hunk 1]
```diff
@@ -82,23 +54,20 @@ buildPythonApplication rec {
...
+    homepage = "https://github.com/jellyfin/jellyfin-mpv-shim";
     description = "Allows casting of videos to MPV via the jellyfin mobile and web app";
+    longDescription = ''
+      Jellyfin MPV Shim is a client for the Jellyfin media server which plays media in the
+      MPV media player. The application runs in the background and opens MPV only
+      when media is cast to the player. The player supports most file formats, allowing you
+      to prevent needless transcoding of your media files on the server. The player also has
+      advanced features, such as bulk subtitle updates and launching commands on events.
+    '';
     license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/video/jellyfin-mpv-shim/default.nix:71]: gpl3Only or gpl3Plus?

Comment (jojosch) [reply pkgs/applications/video/jellyfin-mpv-shim/default.nix:71]: Opened an issue to clarify that - https://github.com/jellyfin/jellyfin-mpv-shim/issues/210

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 119927` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>jellyfin-mpv-shim</li>
</ul>
</details>

File: pkgs/applications/video/jellyfin-mpv-shim/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/video/jellyfin-mpv-shim/default.nix:71]:
```suggestion
    license = with licenses; [ mit gpl3Only ];
```

Comment (jojosch) [reply pkgs/applications/video/jellyfin-mpv-shim/default.nix]: added an explicit listing of all the used licenses by the main application and the shader packs.

**Comment by jojosch** [comment]
Update to latest release https://github.com/jellyfin/jellyfin-mpv-shim/releases/tag/v2.0.1 ([changes](https://github.com/jellyfin/jellyfin-mpv-shim/compare/v2.0.0...v2.0.1))

**Pull request merged**